services/feedback-service/app.py

```diff
@@ -0,0 +1,80 @@
+import os
+import signal
+import sys
+
+import requests
+from cloudevents.conversion import to_binary
+from cloudevents.http import CloudEvent
+from flask import Flask, request
+
+K_SINK = os.environ.get("K_SINK")
+SOURCE_DECLARATION = os.environ.get("SOURCE_DECLARATION")
+
+if not K_SINK:
+    raise Exception("Missing K_SINK")
+if not SOURCE_DECLARATION:
+    raise Exception("Missing SOURCE_DECLARATION")
+
+print(f"Going to send events to K_SINK: {K_SINK}")
+print(f"Going to use CloudEvent source value as '{SOURCE_DECLARATION}'")
+
+
+def handler(signal, frame):
+    print('Gracefully shutting down')
+    sys.exit(0)
+
+
+signal.signal(signal.SIGINT, handler)
+signal.signal(signal.SIGTERM, handler)
+
+app = Flask(__name__)
+
+
+@app.route("/", methods=["POST"])
+def receive_feedback():
+    # request body looks like this:
+    # {
+    #  "uploadId": "123",
+    #  "feedback": "5"
```

Comment:
should this be a number ?
```suggestion
    #  "feedback": 5
```